testdata/cloudflare/cloudflare_filter.yaml

```diff
@@ -22,13 +22,19 @@ interactions:
             "paused": false,
             "description": "Restrict access from these browsers on this address range.",
             "ref": "FIL-100"
+          },
+          {
+            "id": "930d12911fa04ff1924ef5945c232dfc",
+            "expression": "(http.user_agent contains \"${jndi\" or \"%{other}\")",
```

Comment:
missing end bracket here i think.  `\"${jndi}\"`

Reply:
It's intentional because this was the weird rule I found